src/broker/broker.c

```diff
@@ -92,6 +92,8 @@ int broker_new(Broker **brokerp, int log_fd, int controller_fd, uint64_t max_byt
         if (r)
                 return error_fold(r);
 
+        broker->bus.log = &broker->log;
```

Comment:
Now there is a random time between bus_init() and this where bus->log is NULL. This will not work as expected. Why not pass it in to bus_init()?

Reply:
Done.